### PR TITLE
Bug 1838842 - Ship v121.0.0 of the WebCompat System Addon

### PR DESCRIPTION
**Pull request by ksy36**
https://bugzilla.mozilla.org/show_bug.cgi?id=1838842

### CODE REVIEW

File: src/injections/js/bug1864564-esri-transfrom-names-shim.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * https://bugzilla.mozilla.org/show_bug.cgi?id=1855763, this results
+ * in maps partially or incorrectly displayed. Overriding  esri._css.names
+ * containing -moz-prefixed css properties to their unprefixed versions
+ * fixes the issues.
+ */
+
+/* globals exportFunction, cloneInto */
+
+console.info(
+  "esri._css.names for compatibility reasons. See https://bugzilla.mozilla.org/show_bug.cgi?id=1864564 for details."
```
Comment (denschub) [inline src/injections/js/bug1864564-esri-transfrom-names-shim.js:22]: Nit: I think there's a word missing here.

**Review by denschub — APPROVED** [review]
Thank you! 🚀

**Comment by ksy36** [comment]
Thanks :)

**Pull request merged**